lib/global.gd

```diff
@@ -155,6 +170,26 @@ DeclareGlobalFunction("MakeReadWriteGlobal");
 
 #############################################################################
 ##
+#F  MakeConstantGlobal( <name> )   . . . . make a global variable constant
+##
+##  <ManSection>
+##  <Func Name="MakeConstantGlobal" Arg='name'/>
+##
+##  <Description>
+##  MakeConstantGlobal ( <A>name</A> ) marks the global variable named
+##  by the string <A>name</A> as constant. A constant variable can never
+##  be changed or made read-write. Constant variables can only take an
+##  integer value, <C>true</C> or <C>false</C>.
```

Comment:
should that be "immediate integer value"? Hmm, though not everybody will know what that means..

Reply:
I don't think the concept is defined in the documentation -- not anywhere obvious, anyway. Maybe just say that there is a limit on the size of the integers.